lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -310,7 +310,7 @@ function dosomething_user_form_alter(&$form, $form_state, $form_id) {
 
       $form['#after_build'][] = 'dosomething_user_remove_extra_values_from_address_field';
       if ($validate_address) {
-        $form['#after_build'][] = 'dosomething_user_add_validation_attributes_to_address_fields';
+//        $form['#after_build'][] = 'dosomething_user_add_validation_attributes_to_address_fields';
```

Comment:
Oh, yeah this should probably be removed (and we should start doing that in Northstar).